src/wazuh_db/wdb_parser.c

```diff
@@ -12,6 +12,171 @@
 #include "wdb.h"
 #include "external/cJSON/cJSON.h"
 
+static struct column_list const table_hotfixes[] = {
```

Comment:
Please, update this name, and the following ones, to be compliant with the C Coding Style Guide (ALL_CAPITAL_WITH_UNDERSCORES)